### DISCUSSION TITLE
When I execute the command: ./build.sh , compilation failed , the following error occurs

### DISCUSSION THREAD

**Discussion by ddwy** [USC-ACTLab/crazyswarm#390] [Q&A]
Hello,

When I execute the command: ./build.sh ,  compilation failed ,  the following error occurs:

arm-none-eabi-size cf2_nrf.elf
   text	   data	    bss	    dec	    hex	filename
  65440	    612	   5032	  71084	  115ac	cf2_nrf.elf
S110 Activated
BLE  Disabled
Built for platform cf2
  CLEAN_VERSION
  CC    position_controller_indi.o
  CC    cfassert.o
  CC    exptest.o
  CC    cpuid.o
  CC    exptestRR.o
  CC    eprintf.o
  CC    exptestBolt.o
  CC    filter.o
.//src/modules/src/position_controller_indi.c: In function 'positionControllerINDI':
.//src/modules/src/position_controller_indi.c:180:2: error: missing braces around initializer [-Werror=missing-braces]
  float M_OB[3][3] = {0};
  ^
.//src/modules/src/position_controller_indi.c:180:2: error: (near initialization for 'M_OB[0]') [-Werror=missing-braces]
  CC    crc.o
.//src/modules/src/position_controller_indi.c: At top level:
cc1: error: unrecognized command line option "-Wno-address-of-packed-member" [-Werror]
cc1: all warnings being treated as errors
tools/make/targets.mk:26: recipe for target 'position_controller_indi.o' failed
make[1]: *** [position_controller_indi.o] Error 1
make[1]: *** Waiting for unfinished jobs....
Makefile:375: recipe for target 'build' failed
make: *** [build] Error 2

**Comment by jpreiss** [comment]
This error is from building https://github.com/bitcraze/crazyflie2-nrf-firmware. Can you clone that repo by itself and build it? (see https://www.bitcraze.io/documentation/repository/crazyflie-firmware/2020.04/building-and-flashing/build_instructions/, https://github.com/bitcraze/crazyflie2-nrf-firmware/blob/master/docs/build/build.md). If not, resolve that problem first.

Alternatively, you could skip the firmware build and use the prebuilt binaries in https://github.com/USC-ACTLab/crazyswarm/tree/master/prebuilt. You can comment out the "build *** firmware" steps in `build.sh`.
